### PR TITLE
feat(ResultsListItem): add clickable keywords

### PR DESCRIPTION
**Pull request by Delapouite**

### CODE REVIEW

**Comment by satazor** [comment]
Instead of relying on javascript click event, can you change to actually use the anchor href?

**Comment by Delapouite** [comment]
Yes. Done.

File: src/routes/search/components/ResultsListItem.js [hunk 1]
```diff
@@ -76,6 +88,7 @@ class ListItem extends Component {
     _onGravatarLoad(e) {
         e.target.style.opacity = '1';
     }
+
 }
```
Comment (satazor) [inline src/routes/search/components/ResultsListItem.js:91]: Can you remove this line?

File: src/routes/search/components/ResultsListItem.js [hunk 2]
```diff
@@ -50,6 +45,23 @@ class ListItem extends Component {
...
+        if (!keywords) {
+            return '';
+        }
+
+        return (
+            <div className="keywords ellipsis">
+                <MaterialIcon id="local_offer" />
+                { keywords.map((keyword) =>
+                    <a key={ keyword } href={ '/search?term=' + keyword }>{ keyword }</a>
+                ) }
```
Comment (satazor) [inline src/routes/search/components/ResultsListItem.js:60]: Better if we wrap the keyword in encodeURIComponent just to be safe

**Comment by satazor** [comment]
The PR is looking good!  Can you apply my second rouund of suggestions? I'll merge it then.

**Comment by Delapouite** [comment]
Ok. Should be better now.

**Comment by satazor** [comment]
Ty @Delapouite, will deploy tomorrow!

**Pull request merged**